tutorials/R/R-skills/Colorado-floods-data-visualization/NEON-Boulder-Flood-LiDAR-in-R/NEON-Boulder-Flood-LiDAR-in-R.Rmd

```diff
@@ -342,7 +358,7 @@ cropbox2<-c(473792.6,474999,4434526,4435453)
 Once you have the crop box defined, either manually or using the coordinate, you 
 then crop the desired layer to the crop box. 
 
-```{r plot-crop-raster}
+```{r plot-crop-raster, fig.cap=c(" Raster Plot of the cropped section of Four Mile Creek, Boulder County.", "Raster Plot of the cropped section of Four Mile Creek, Boulder County, Post-Flood.","Plot of the Elevation change,Post-flood,in the cropped section of Four Mile Canyon Creek, Boulder County with elevation change represented in categories (breaks).")}
```

Comment:
Can remove the space at the beginning of the first fig.cap quote, and add spaces around "post-flood"